samples/net/secure_mqtt_sensor_actuator/README.rst

```diff
@@ -0,0 +1,219 @@
+.. zephyr:code-sample:: secure-mqtt-sensor-actuator
+   :name: Secure MQTT Sensor/Actuator
+   :relevant-api: mqtt_socket sensor_interface
+
+   Implement an MQTT-based IoT sensor/actuator device
+
+Overview
+********
+
+This sample demonstrates the implementation of an IoT sensor/actuator device.
+The application uses the MQTT protocol to securely send sensor data
+to a remote MQTT broker, while responding to commands received over MQTT.
+
+Features:
+
+- Establishing network connectivity using a DHCPv4 lease
+- Establishing a secure MQTT connection (using TLS 1.2) to MQTT broker
+- Publishing temperature sensor data in JSON format to the MQTT broker at a user-defined interval
+- Subscribing to user-defined topic(s) on MQTT broker
+- Responding to commands received over the network (LED control)
+- Handling of MQTT connection, re-connecting and keep-alive
+- Network status LED
+
+Requirements
+************
+- Board with network capability (tested with adi_eval_adin1110ebz)
+- `Eclipse Mosquitto`_ MQTT broker
+- DHCP server
+- Network connection between the board and Mosquitto broker
+
+Build and Running
+*****************
+This application relies on an network connection between the board and an MQTT broker.
+This broker can exist locally (e.g. on a host PC) or a publicly available MQTT broker
+<https://test.mosquitto.org/> can be used.
+For quick sampling/testing, a configuration is provided to connect to a local MQTT broker
+without security, using a static IP address.
+
+Hardware Setup
+==============
+If using Ethernet, connect the board to the MQTT broker. This may be your host PC
+(for locally hosted Mosquitto broker) or your internet router
+(to connect to the public Mosquitto broker).
+If required, connect a temperature sensor to the board.
+
+Software Setup
+==============
+The temperature sensor should be aliased in devicetree as ``ambient-temp0``.
+If a board does not include an on-board temperature sensor, one can be connected externally
+and a board overlay file used to add the sensor and alias:
+
+.. code-block:: devicetree
+
+	/ {
+		aliases {
+				ambient-temp0 = &adt7420;
+			};
+		};
+	};
+
+It is possible to use other types of sensors, by adding them in devicetree and by changing
+``SENSOR_CHAN`` :file:`in device.c` to match the desired sensor type.
+
+There are a few ways to configure the application:
+
+.. list-table::
+
+		* - :file:`prj.conf`
+			- Default config: Secure MQTT, dynamic IP address (DHCP)
+
+		* - :file:`overlay-static.conf`
+			- Secure MQTT, static IP address
+
+		* - :file:`overlay-static-insecure.conf`
+			- Unsecure MQTT, static IP address
```

Comment:
this is not correctly formatted (see [output doc](http://127.0.0.1:8080/samples/net/secure_mqtt_sensor_actuator/README.html#software-setup)) , probably due to incorrect tabulations

```suggestion
.. list-table::

   * - :file:`prj.conf`
     - Default config: Secure MQTT, dynamic IP address (DHCP)

   * - :file:`overlay-static.conf`
     - Secure MQTT, static IP address

   * - :file:`overlay-static-insecure.conf`
     - Unsecure MQTT, static IP address
```